routing_common/transit_types.hpp

```diff
@@ -133,6 +130,8 @@ class TitleAnchor
 class Stop
 {
 public:
+  NEWTYPE(StopId, MonotoneStopId);
```

Comment:
Please, get rid of "Monotone" prefix. The datatype does not (and can't) ensure that all values in all possible sequences will be monotone. Only you can ensure this. So, "Monotone" prefix is wrong and misleading here.

Reply:
Yes. I fill the data the way they're monotone. On the other hand I can't just remove the prefix (StopId is not free). Let's think about a better prefix/suffix. 

`WrappedStopId`?

Reply:
I don't know ) Why we can't use StopId?

Reply:
The name StopId (using StopId = uint64_t) is busy. It's used in several data structures (Edge, Shape, Stop) in transit sub system.

Reply:
Done